src/callback_handler/sigstore_verification.rs

```diff
@@ -57,9 +72,24 @@ impl Client {
         if keyless.is_empty() {
             return Err(anyhow!("Must provide keyless info"));
         }
+        // Build intering VerificationConfig:
```

Comment:
Really silly comment, maybe it's a limitation of my English... but I don't get the meaning of this comment. :sweat_smile:

Reply:
I believe @viccuad wrote this method (I just moved it from the fetcher to the evaluator) I think it means intermediate or auxiliary VerificationConfig, right Victor?

Reply:
yes, that's a typo, it should be "interim".

Reply:
Could be dropped for symmetry, it's not present in `verify_public_key()`.

Reply:
do as you wish, I don't have any preference :)